_posts/2024-07-19-Using-Redis-with-Sidecar.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "Leveraging Redis as a Sidecar for Linux App Service"
+author_name: "Tulika Chaudharie"
+toc: true
+toc_sticky: true
+---
+
+At Microsoft Build 2024, we had announced the [Public Preview for the Sidecar pattern for Linux App Service](https://azure.github.io/AppService/2024/04/04/Public-Preview-Sidecars-Webjobs.html). This feature allows developers to enhance their web applications with additional services running alongside the main application container. In this blog, we will demonstrate how you can leverage the Sidecar pattern to seamlessly integrate a Redis cache into your Linux App Service, boosting your application's performance and reliability.
+
+## Setting Up your local environment
+
+Before you deploy your application to Azure App Service, you can set it up locally by following these steps
+
+1. **Prerequisites:** Ensure you have [Docker Desktop](https://www.docker.com/products/docker-desktop/) and Visual Studio 2019 or above installed on your machine.
```

Comment:
do I really need visual studio for this? can't I use vscode? Keep in mind the VS implies windows, VSCODE can be used in Linux/MAC/Win...

Reply:
Done